cgi-bin/LJ/User/Account.pm

```diff
@@ -12,8 +12,11 @@
 # part of this distribution.
 
 package LJ::User;
+
 use strict;
-no warnings 'uninitialized';
```

Comment:
Yeah that's probably code we should fix, yeah? I fixed one of them in the RPC popup stuff. Do you think it's too much/we shouldn't do that?